docs/topics/request-response.rst

```diff
@@ -508,6 +508,38 @@ method for this job. Here's an example spider which uses it::
 
             # continue scraping with authenticated session...
 
+JSONRequest
+-----------
+
+The JSONRequest class extends the base :class:`Request` class with functionality for
+dealing with JSON requests.
+
+.. class:: JSONRequest(url, [data, ...])
+
+   The :class:`JSONRequest` class adds a new argument to the constructor called data. The
+   remaining arguments are the same as for the :class:`Request` class and are
+   not documented here.
+
+   Using the :class:`JSONRequest` will set the `Content-Type` header to `application/json`
```

Comment:
Single ticks \` will be rendered in italic, while double \`` will be rendered as ``code``, and I think in this line and below it's better to use double quotes.
```suggestion
   Using the :class:`JSONRequest` will set the ``Content-Type`` header to ``application/json``
```

Reply:
Changed.